crates/rapier3d-urdf/Cargo.toml

```diff
@@ -21,6 +27,7 @@ anyhow = "1"
 bitflags = "2"
 # NOTE: we are not using the (more recent) urdf-rs crate because of https://github.com/openrr/urdf-rs/issues/94
 xurdf = "0.2"
+profiling = "1.0"
```

Comment:
I added the `profiling` dependency there but I think I didn´t mark any functions